### DISCUSSION TITLE
Will an empty node be deleted by papermerge

### DISCUSSION THREAD

**Discussion by jalwn** [papermerge/papermerge-core#178] [Q&A]
Hello, I plan to use papermerge to store a lot of documents. If there are documents older than 6 months I would like to remove them from the filesystem and archive them somewhere else (maybe compress them) so that I wont have to keep on increasing storage for papermerge. Is there any way to do this archiving of old documents with papermerge? Any help will be greatly appreciated. Thank you!

**Comment by ciur** [comment]
yes, you can use REST API for that. For that you need to write a REST API client, which will run, e.g. once a day and check the created_at field of all documents. If that create_at field is older then e.g. 6 months, then again, via REST API you can download that document/specific version of the document (e.g. initial version) and then do whatever you want with it e.g. move to separate folder.

You can see the full REST API interface by starting web app with (for example):

$ docker run -p 9400:8000 \
    -e PAPERMERGE__MAIN__SECRET_KEY=abc \
    -e DJANGO_SUPERUSER_PASSWORD=123 \
    papermerge/papermerge:2.1.9


Then open you browser at http://localhost:9400, username=admin, password=123


![rest-api](https://github.com/papermerge/papermerge-core/assets/24827601/600d2033-765b-4667-9f73-16a46d6c0530)
